content/docs/concepts/traffic-management/index.md

```diff
@@ -133,7 +133,7 @@ Istio does not provide a DNS. Applications can try to resolve the
 FQDN using the DNS service present in the underlying platform (`kube-dns`,
 `mesos-dns`, etc.).
 
-### Ingress and egress
+### Ingress and Egress
```

Comment:
Should we lowercase e, we only capitalize the first letter of headings.

Reply:
I read the style guide and I agree.